pkg/restore/util.go

```diff
@@ -309,6 +310,9 @@ func matchNewPrefix(key []byte, rewriteRules *RewriteRules) *import_sstpb.Rewrit
 }
 
 func truncateTS(key []byte) []byte {
+	if len(key) == 0 {
```

Comment:
Did this fix `panic: runtime error: slice bounds out of range [:-8]`?

Reply:
This fix the panic but it seems the `key` being nil itself is a bug. We can remove this change.